cg/models/demultiplex/flow_cell.py

```diff
@@ -210,10 +211,19 @@ def validate_flow_cell_dir_name(self) -> None:
             LOG.warning(message)
             raise FlowCellError(message)
 
-    def is_demultiplexing_started(self) -> bool:
-        """Check if demultiplexing started path exists."""
+    def has_demultiplexing_started_locally(self) -> bool:
+        """Check if demultiplexing has started path exists on the cluster."""
         return self.demultiplexing_started_path.exists()
 
+    def has_demultiplexing_started_on_deck(self) -> bool:
+        """Check if demultiplexing has started on the NovaSeq x machine."""
+        latest_analysis: Path = get_latest_analysis_path(self.path)
+        if not latest_analysis:
+            return False
+        return Path(
+            latest_analysis, DemultiplexingDirsAndFiles.DATA, DemultiplexingDirsAndFiles.BCL_CONVERT
+        ).exists()
```

Comment:
```suggestion
        latest_analysis: Path = get_latest_analysis_path(self.path)
        if not latest_analysis:
            return False
        return Path(
            latest_analysis, DemultiplexingDirsAndFiles.DATA, DemultiplexingDirsAndFiles.BCL_CONVERT
        ).exists()
```

Is this not checking locally whether the demultiplexing has finished on the sequencer?
I believe there is a "SecondaryAnalysisComplete.txt" file when these analysis are finished written. Maybe that is a better indicator? I think the enum is: `ANALYSIS_COMPLETED`

Reply:
The problem is that there might be a time window where sequencing is complete but demultiplexing (on deck) is not, so we want to know whether it is running at all, thus looking for the `BCLConvert` folder.